src/api/sendgrid.ts

```diff
@@ -0,0 +1,53 @@
+import mail, { MailDataRequired } from '@sendgrid/mail'
+import apiEnv from './apiEnv';
+
+export interface Personalizations {
```

Comment:
Why not use sendgrid's native definition of PersonalizationData? https://github.com/sendgrid/sendgrid-nodejs/blob/b8125d86134157a462c2e24816cfa56677ab76a8/packages/helpers/classes/personalization.d.ts#L3